genesis/src/main.rs

```diff
@@ -333,11 +333,19 @@ fn main() -> Result<(), Box<dyn error::Error>> {
     let bootstrap_storage_pubkey = pubkey_of(&matches, "bootstrap_storage_pubkey_file");
     let faucet_pubkey = pubkey_of(&matches, "faucet_pubkey_file");
 
-    let rent = Rent {
-        lamports_per_byte_year: value_t_or_exit!(matches, "lamports_per_byte_year", u64),
-        exemption_threshold: value_t_or_exit!(matches, "rent_exemption_threshold", f64),
-        burn_percent: value_t_or_exit!(matches, "rent_burn_percentage", u8),
-    };
+    let rent_burn_percentage = value_t_or_exit!(matches, "rent_burn_percentage", u8);
+    if rent_burn_percentage > 100 {
+        panic!(
```

Comment:
Let's never panic on user input, instead provide a clean exit for the user to retry.   In this case, a clap.rs `.validator()` attached to the rent_burn_percentage arg would be ideal, eg: 
https://github.com/solana-labs/solana/blob/d5c8b26a458bc469229640b359aa05339ca66c38/gossip/src/main.rs#L69

Reply:
cool. Will do that.